README

```diff
@@ -87,3 +87,8 @@ prior to distribution:
   packagesbuild resources/neoman.pkgproj
   productsign --sign 'Developer ID Installer' dist/YubiKey\ NEO\ Manager.pkg dist/yubikey-neo-manager-mac.pkg
 
+
+==== Linux
```

Comment:
This part is under the "Building binaries" section, which uses PySide to build standalone executables that bundle all dependencies. We haven't tested this for Linux, mainly because there shouldn't be any reason to need to do so on Linux. So I'm not sure what this part adds. For Linux the recommended approach is to use the provided Debian package (for Debian or Ubuntu) or install via pip, which will take care of the needed python dependencies.